charts/osm/crds/meshconfig.yaml

```diff
@@ -187,6 +187,14 @@ spec:
                       description: Sets the service certificate validity duration, represented as a sequence of decimal numbers each with optional fraction and a unit suffix.
                       type: string
                       default: "24h"
+                experimental:
+                  description: Experimental configurations
+                  type: object
+                  properties:
+                    clusterID:
```

Comment:
Maybe call this clusterName or clusterDomain

Reply:
Renamed to clusterDomain along with associated properties/methods